src/axom/spin/internal/linear_bvh/emit_bvh.hpp

```diff
@@ -106,21 +106,21 @@ void emit_bvh(RadixTree<FloatType, 3>& data, BVHData<FloatType, 3>& bvh_data)
                                // do the offset now
                                rchild *= 4;
                              }
-                             vec1[0] = l_aabb.m_x.min();
-                             vec1[1] = l_aabb.m_y.min();
-                             vec1[2] = l_aabb.m_z.min();
+                             vec1[0] = l_aabb.getMin()[0];
```

Comment:
Could this use something like:
```cpp
primal::Vector<FloatType, 4> vec1(l_aabb.getMin().array());
```
or
```cpp
primal::Vector<FloatType, 4> vec1 = l_aabb.getMin().array();
```
(and similarly for the others)

Or is it better to explicitly set each entry within the kernel for some reason?

Reply:
The code here packs the 3D min/max points from two bounding boxes into 4-wide vectors. We could do a direct assignment for `l_aabb.getMin() -> vec1`, but the other points are split up between two of the vec4.

Reply:
Thanks.  You're right, I wasn't paying attention to the indexing.